src/main/java/edu/cornell/kfs/fp/document/service/CuDisbursementVoucherTaxService.java

```diff
@@ -0,0 +1,11 @@
+package edu.cornell.kfs.fp.document.service;
+
+public interface CuDisbursementVoucherTaxService {
+	
+	public boolean isForeignVendorWithTaxReviewRequired(String payeeTypeCode,
+			String paymentReasonCode, Integer vendorHeaderId);
+	
+	public boolean isForeignVendoAndNOTaxReviewRequired(String payeeTypeCode,
```

Comment:
Does the additional 'Is' in the proposed method name necessary?  It seems grammatically incorrect, does not flow right in my mind.